src/classmentors/components/events/events-view-event-task-form.html

```diff
@@ -95,10 +95,29 @@
         <div>
             <h3>Task type <small ng-if="!ctrl.taskType">(Required)</small></h3>
             <md-radio-group ng-model="ctrl.taskType">
-                <md-radio-button value="service" class="md-primary">Service / Badge / Problem</md-radio-button>
-                <md-radio-button value="linkPattern" class="md-primary">Link pattern</md-radio-button>
-                <md-radio-button value="textResponse" class="md-primary">Text Response</md-radio-button>
-                <md-radio-button value="indexCard" class="md-primary">Index Card</md-radio-button>
+                <!--<md-radio-button value="service" class="md-primary">Service / Badge / Problem</md-radio-button>-->
+                <!--<md-radio-button value="linkPattern" class="md-primary">Link pattern</md-radio-button>-->
+                <!--<md-radio-button value="textResponse" class="md-primary">Text Response</md-radio-button>-->
+                <!--<md-radio-button value="indexCard" class="md-primary">Index Card</md-radio-button>-->
+
+                <table>
+                    <tr>
+                        <td>
+
+                            <md-radio-button value="service" class="md-primary">Service / Badge / Problem</md-radio-button>
+                            <md-radio-button value="linkPattern" class="md-primary">Link pattern</md-radio-button>
+                            <md-radio-button value="textResponse" class="md-primary">Text Response</md-radio-button>
+                            <md-radio-button value="multipleChoice" class="md-primary">Multiple Choice</md-radio-button>
+                            <md-radio-button value="indexCard" class="md-primary">Index Card</md-radio-button>
+                        </td>
+                        <td style='padding-left:20px; padding-bottom:40px;'>
+
+                            <md-radio-button value="code" class="md-primary">Code</md-radio-button>
+                            <md-radio-button value="video" class="md-primary">Video</md-radio-button>
+                            <md-radio-button value="journalling" class="md-primary">Journalling</md-radio-button>
+                        </td>
+                    </tr>
+                </table>
```

Comment:
@ceocookie @shiquan-lim 

Can you please use ng-material layout features instead of a table?

[Layout docs](https://material.angularjs.org/1.0.9/layout/introduction)
[Form example](https://material.angularjs.org/1.0.9/demo/input)
[Radio Button docs](https://material.angularjs.org/1.0.9/demo/radioButton)